mne_bids/tests/test_write.py

```diff
@@ -1376,7 +1376,7 @@ def test_write_anat(_bids_validate):
                    overwrite=False)
 
     # pass some invalid type as T1 MRI
-    with pytest.raises(ValueError, match='must be a path to a T1 weighted'):
+    with pytest.raises(ValueError, match='must be a path to a MRI'):
```

Comment:
```suggestion
    with pytest.raises(ValueError, match='must be a path to an MRI'):
```

Reply:
needed if @hoechenberger's [suggestion here](https://github.com/mne-tools/mne-bids/pull/641/files#r530878959) is accepted